DigitalOffice/Services/AuthenticationService/Token/JwtValidation.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.IdentityModel.Tokens.Jwt;
+using LT.DigitalOffice.AuthenticationService.Token.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+
+namespace LT.DigitalOffice.AuthenticationService.Token
+{
+    public class JwtValidation : IJwtValidation
```

Comment:
JwtValidator will be better (not nessesary)

Reply:
Fixed